src/pages/Admin/Charity/EditProfile/schema.ts

```diff
@@ -23,7 +23,15 @@ const fileObj = Yup.object().shape<SchemaShape<ImgLink>>({
 //construct strict shape to avoid hardcoding shape keys
 const shape: SchemaShape<FormValues> = {
   categories_sdgs: Yup.array().min(1, "required"),
-  tagline: requiredString,
+  tagline: Yup.string().test(
```

Comment:
@SovereignAndrey, came across `lint` and `type` errors when spinning up this PR 
created fix #1790